src/coreclr/vm/fieldmarshaler.h

```diff
@@ -24,7 +24,7 @@ class MethodTable;
 // Currently we set this to the packing size of the largest supported
 // fundamental type and let the field marshaller downsize where needed.
 //=======================================================================
-#define DEFAULT_PACKING_SIZE 32
+#define DEFAULT_PACKING_SIZE 64
```

Comment:
This is one of those scary constants that is easily missed. Can we make a comment that this is now 64 bytes due to the 512 type?

Reply:
Can do 👍